src/components/link/link.component.js

```diff
@@ -156,12 +157,14 @@ Link.propTypes = {
   /** A routing component to render when the to prop is set */
   routerLink: PropTypes.oneOfType([PropTypes.func, PropTypes.object]),
   /** Target property in which link should open ie: _blank, _self, _parent, _top */
-  target: PropTypes.string
+  target: PropTypes.string,
+  /** Ref to be forwarded */
+  innerRef: PropTypes.object
```

Comment:
Link component breaks because of story.js and Component.defaultProps, I have added forwardRef to Link what causes problems with storybook and actions, but if you type props without Component.defaultProps then it works good

Reply:
I think we still need to mark `innerRef` as `@private` and `@ignore`.

Reply:
I think we still need to mark `innerRef` as `@private` and `@ignore`.

Reply:
It is updated, you will need to choose "show all changes" option probably to see it